extension/content.js

```diff
@@ -344,6 +344,21 @@ function addFilterCommentsByYou() {
 	}
 }
 
+function addProjectNewLink() {
+	const projectNewLinkId = 'refined-github-project-new-link';
+	const projectNewLink = `<a href="/${repoUrl}/projects/new" class="btn btn-sm" id="${projectNewLinkId}">Add a project</a>`;
+	if ($(`#${projectNewLinkId}`).length === 0) {
```

Comment:
> Just one more thing that came to my mind: what about the case when a user has some projects already? Shall we still have the button shown in the Settings page?

I don't see why not, it will be most handy to people using Projects already anyway.